bundle.md

```diff
@@ -8,17 +8,22 @@ See also [MacOS application bundles][macos_bundle] for a similar use of the term
 The definition of a bundle is only concerned with how a container, and its configuration data, are stored on a local filesystem so that it can be consumed by a compliant runtime.
 
 A Standard Container bundle contains all the information needed to load and run a container.
-This MUST include the following artifacts:
+This includes the following artifacts:
 
 1. <a name="containerFormat01" />`config.json`: contains configuration data.
 This REQUIRED file MUST reside in the root of the bundle directory and MUST be named `config.json`.
 See [`config.json`](config.md) for more details.
 
 2. <a name="containerFormat02" />A directory representing the root filesystem of the container.
-While the name of this REQUIRED directory may be arbitrary, users should consider using a conventional name, such as `rootfs`.
-This directory MUST be referenced by [`root`](config.md#root) within the `config.json` file.
+While the name of this directory may be arbitrary, users should consider using a conventional name, such as `rootfs`.
 
-While these artifacts MUST all be present in a single directory on the local filesystem, that directory itself is not part of the bundle.
+On non-Windows platforms, this directory is REQUIRED.
+
+On Windows, for Windows Server containers, this directory is REQUIRED. For Hyper-V containers, it MUST be omitted.
+
+If set, this directory MUST be referenced by [`root`](config.md#root) within the `config.json` file.
```

Comment:
I think these are all part of the second list entry.  To get multi-paragraph list entries in Pandoc, you need four-space (or tab) indents (more in #495).  Can you use:

```
2. <a name="containerFormat02" />A directory representing the root filesystem of the container.
    While the name of this directory may be arbitrary, users should consider using a conventional name, such as `rootfs`.
 
    On non-Windows platforms, this directory is REQUIRED.

    On Windows, for Windows Server containers, this directory is REQUIRED.
    For Hyper-V containers, it MUST be omitted.

    If set, this directory MUST be referenced by [`root`](config.md#root) within the `config.json` file.
```